en/0-intro.md

```diff
@@ -4,7 +4,7 @@
 
 Today’s work market lacks efficiency because of high transaction costs[^1][^2][^3] and frequent conflicts of interests between the employer and the employee.
 
-The WoM (World of Meeds) proposes a new paradigm that would lower transaction costs and reinvent the employment relationship, bridging the gap between traditional corporations and DAOs.
+The WoM (World of Meeds) proposes a new paradigm that would lower transaction costs and reinvent the employment relationship, bridging the gap between traditional corporations and DAOs (Decentralized Autonomous Organizations).
```

Comment:
I don't think putting the full meaning of DAOs in bracket is necessary. Considering the fact that it a whitepaper and definition of terms can be done at the start or end of the paper. I think every abbreviation should be define at the start of the paper.

Reply:
Thanks. I plan to include a glossary of terms, but I still think it's convenient for the reader to have them explained in parentheses on the first occurrence. This way he doesn't have to jump to the glossary every time.

Reply:
The abbreviation is what is supposed to be in parentheses, not the full meaning of it.